draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -984,9 +987,9 @@ PNCs:
 
 - the network topology, at both optical and IP layers, in terms of
 nodes and links, including the access links, inter-domain IP links
-as well as cross-layer links;
+as well as Cross-technology links;
```

Comment:
Should be a "Cross-technology **Ethernet** links" as per new terminology, correct?

Reply:
```suggestion
as well as cross-technology Ethernet links;
```